src/main/main.c

```diff
@@ -311,8 +311,14 @@ void init(void)
     spiInit(SPIDEV_1);
 #endif
 #ifdef USE_SPI_DEVICE_2
+#ifdef OMNIBUSF4
```

Comment:
How about creating a sub-target, say, OMNIBUSF4V2 for now, and consolidate OMNIBUSF4 and OMNIBUSF4V2 later if possible?

I have a feeling that resource assignments mechanism, when completed, will take care most of the problems we have with the V2 board, including timers, uarts and i2c.